x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/attribute_selector.tsx

```diff
@@ -103,13 +112,13 @@ export const AttributeSelector: React.FC<IAttributeSelectorProps> = ({
       className={disabled ? 'euiPanel--disabled' : ''}
     >
       <EuiTitle size="s">
-        <h3>Attribute mapping</h3>
+        <h3>{ATTRIBUTE_SELECTOR_TITLE}</h3>
```

Comment:
The original copy says `Attribute mapping`. Why we use a different name for a variable?

Reply:
We typically totle the constants by their use. This component is the `AttributeSelector` and this is its title, so I used that as the variable name.